### PR TITLE
ENH: FileStore integrated reader

### PR DESCRIPTION
**Pull request by chiahaoliu**
This short PR is not in the original release scope but I found this change could a complete cycle possible, from generating images, pushing to local ``db`` and querying them. This can potentially make our tests a lot more robust.

@sbillinge @CJ-Wright PR is ready for review

### CODE REVIEW

File: xpdacq/simulation.py [hunk 1]
```diff
@@ -190,6 +180,7 @@ def insert_imgs(mds, fs, n, shape, save_dir=tempfile.mkdtemp()):
 
 # instantiate simulation objects
 db = build_pymongo_backed_broker()
-pe1c = SimulatedPE1C('pe1c', {'pe1_image': lambda: 5})
+db.fs.register_handler('RWFS_NPY', be.ReaderWithFSHandler)
+pe1c = SimulatedPE1C('pe1c', {'pe1_image': lambda: np.ones((5,5))}, fs=db.fs)
```
Comment (CJ-Wright) [inline xpdacq/simulation.py]: If you want you can steal the image cycle logic from [xpdSim](https://github.com/xpdAcq/xpdSim/pull/13/files#diff-a2f91388ba37eebfc35980ace3307f21R15) and the use of a data directory to use real images.

Comment (chiahaoliu) [inline xpdacq/simulation.py]: There is the plan for fully integrated with ``xpdSim`` (issue #352) as we discussed a while ago. But I would prefer to do it in separate PR to minimize potential effects.

**Comment by chiahaoliu** [comment]
@CJ-Wright do you have any comment on this PR except for the image logic?

**Comment by CJ-Wright** [comment]
Nope it looks good. Merging.
